src/components/Navbar.tsx

```diff
@@ -1,3 +1,9 @@
+import cn from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+const isActiveLink = ({ isActive }: { isActive: boolean }) =>
```

Comment:
The `isActiveLink` function is expecting an object with a property `isActive` and returns a string. However, when you are using this function in the `NavLink` component (lines 17 and 21), you are not passing any argument. This might lead to a bug.